web-sdk/v3.md

```diff
@@ -0,0 +1,341 @@
+            <span><code>custom</code></span>
+        </dt>
+        <dd>object that allows you to customize the UI for the Web SDK. You can find the structure for it in the customization section below.</dd>
+    </div>
+</dl>
+
+#### Fidel Web SDK Global Object
+
+After adding the Web SDK script on your website, a global variable `Fidel` is created with two methods that you can use to open and close the web form manually, `Fidel.openForm()` and `Fidel.closeForm()`. 
+
+To open the iframe overlay form with a button, you could use this example:
+
+```html
+<button id="open-form" type="submit">
+  Link Card
+</button>
+
+<script type="text/javascript" data-id="fidel-sdk" src="https://resources.fidel.uk/sdk/js/v3/fidel.js">
+</script>
+
+<script type="text/javascript">
+document.getElementById("open-form").addEventListener("click", () => {
+  Fidel.openForm({
+    companyName: "Fidel",
+    sdkKey: "pk_test_demo",
+    programId: "bca59bd9-171b-4d1f-92af-4b2b7305268a",
+    callback() {
+      console.log(arguments);
+    },
+  });
+});
+</script>
+```
+
+##### Web SDK metadata
+
+The metadata `id` property is a *non-unique index*, so you can set it to a custom UID (unique identifier). Later you can query the cards using the same `metadata.id`. You can use our ["List Cards from Metadata ID"](https://reference.fidel.uk/v1/reference#list-cards-from-metadata-id) API Endpoint.
+
+> Hint: Adding user data in the metadata as key: value pairs can simplify reconciliation with your system. For example, adding `'myUserId':'123'` will help you match the added card to your user with id 123.
+
+```html
+fileName:index.html
+<button id="open-form" type="submit">
+  Link Card
+</button>
+
+<script type="text/javascript" data-id="fidel-sdk" src="https://resources.fidel.uk/sdk/js/v3/fidel.js">
+</script>
+
+<script type="text/javascript">
+document.getElementById("open-form").addEventListener("click", () => {
+  Fidel.openForm({
+    companyName: "Fidel",
+    sdkKey: "pk_test_demo",
+    programId: "bca59bd9-171b-4d1f-92af-4b2b7305268a",
+    callback() {
+      console.log(arguments);
+    },
+    metadata: {
+        id: 'this-is-the-metadata-id',
+        myUserId: '123',
+        customKey: 'customValue'
+    }
+  });
+});
+</script>
+```
+
+## Customising the Fidel Web SDK
+
+The `custom` property allows for customising most of the Fidel Web SDK UI. It supports the same CSS properties as the [HTML DOM Style Object](https://www.w3schools.com/jsref/dom_obj_style.asp). Here is a complete list of paramets, all of which are optional.
+
+```javascript
+Fidel.openForm({
+    ...
+    custom: {
+        logo: {
+            url: string,
+            style: CSSPropertiesObject,
+        },
+        form: {
+            style: CSSPropertiesObject,
+        },
+        overlay: {
+            style: CSSPropertiesObject,
+        },
+        close: {
+            location: string, // one of 'form' | 'overlay' | 'none'
+            color: string, // or object
```

Comment:
I'd use the proper type instead of string. And for the `color` it's important to document the exact interface that it can accept other than `string` if not it won't work:
```
location: 'form' | 'overlay' | 'none';
color: 'string' | { form: string; overlay: string };  
```

Reply:
Thanks!